exec/plugin-commands-script-runners/src/run.ts

```diff
@@ -224,10 +224,16 @@ so you may run "pnpm -w run ${scriptName}"`,
     })
   }
   const concurrency = opts.workspaceConcurrency ?? 4
+
+  const extraEnv = {
+    ...opts.extraEnv,
+    ...(opts.rawConfig['node-options'] ? { NODE_OPTIONS: opts.rawConfig['node-options'] } : {}),
```

Comment:
why not read it from `opts.nodeOptions`?

Reply:
It's from the `.npmrc` file too?